Super Mario land 2: implement new game (#2730)
ArchipelagoMW/Archipelago: worlds/marioland2/docs/setup_en.md

# Setup Guide for Super Mario Land 2: 6 Golden Coins

## Important

As we are using BizHawk, this guide is only applicable to Windows and Linux systems.

## Required Software

- BizHawk: [BizHawk Releases from TASVideos](https://tasvideos.org/BizHawk/ReleaseHistory)
  - Version 2.9.1 is recommended.
  - Detailed installation instructions for BizHawk can be found at the above link.
  - Windows users must run the prereq installer first, which can also be found at the above link.
- The built-in Archipelago client, which can be installed [here](https://github.com/ArchipelagoMW/Archipelago/releases)
- A Super Mario Land 2: 6 Golden Coins version 1.0 ROM file. The Archipelago community cannot provide this.

## Configuring BizHawk

Once BizHawk has been installed, open EmuHawk and change the following settings:

- Under Config > Customize > Advanced, make sure the box for AutoSaveRAM is checked, and click the 5s button.
  This reduces the possibility of losing save data in emulator crashes.
- Under Config > Customize, check the "Run in background" box. This will prevent disconnecting from the client while
EmuHawk is running in the background.

> It is strongly recommended to associate GB rom extensions (\*.gb) to the EmuHawk we've just installed.

Reviewer: ```suggestion
It is strongly recommended to associate Game Boy ROM extensions (\*.gb) to the EmuHawk we've just installed.
```

> To do so, we simply have to search any Gameboy rom we happened to own, right click and select "Open with...", unfold

Reviewer: ```suggestion
To do so, we simply have to search any Game Boy ROM we happened to own, right click and select "Open with...", unfold
```

the list that appears and select the bottom option "Look for another application", then browse to the BizHawk folder
and select EmuHawk.exe.

## Configuring your YAML file

### What is a YAML file and why do I need one?

Your YAML file contains a set of configuration options which provide the generator with information about how it should
generate your game. Each player of a multiworld will provide their own YAML file. This setup allows each player to enjoy
an experience customized for their taste, and different players in the same multiworld can all have different options.

### Where do I get a YAML file?

> You can generate a yaml or download a template by visiting the [Super Mario Land 2 Player Settings Page](/games/Super%20Mario%20Land%202/player-settings)

Reviewer 2: ```suggestion
You can generate a yaml or download a template by visiting the [Super Mario Land 2 Player Options Page](/games/Super%20Mario%20Land%202/player-options)
```

## Joining a MultiWorld Game

### Generating and Patching a Game

> 1. Create your settings file (YAML).

Reviewer 2: ```suggestion
1. Create your options file (YAML).
```
